README.md

```diff
@@ -16,12 +16,12 @@ The issue can be tracked [here](https://github.com/vmware-tanzu/cartographer/iss
 Detailed documentation for Cartographer can be found in the `site` folder of this repository:
 
 * [About Cartographer](site/content/docs/about.md): Details the design and philosophy of Cartographer
-* [Examples](examples/source-to-knative-service/README.md): Contains an example of using Cartographer to create a supply chain that takes a repository, creates an image and deploys it to a cluster
+* [Examples](examples/source-to-gitops/README.md): Contains an example of using Cartographer to create a supply chain that takes a repository, creates an image and deploys it to a cluster
```

Comment:
Should we now mention both examples in the readme (i.e. the source-to-gitops portion and the gitops-to-app portion)?

Reply:
Yes, this will become a third example in the repo after a basic deploy and a basic deploy with testing.